### ISSUE TITLE
Arc Welder Processing causes current print to stutter

### ISSUE THREAD

**Issue by BenKitesurfs** [FormerLurker/ArcWelderPlugin#95]
I just installed arc welder last night, and I'm very impressed with it.  It seems to have drastically improved the smoothness of the movements and the final print from my Ramps 1.4 board.

The issue I have noticed so far though, is that if I upload gcode while a print is running, the arc-welding process demands too much processing power from the Pi, and the print that is currently running starts to stutter.

Perhaps this could be fixed by putting uploaded gcode into a cache that will run when the printer is not actively printing

**Comment by FormerLurker** [comment]
Yes, this has been implemented already.  It's taken me MUCH longer than I expected to push out an update, mostly because of some unexpected issues with the enhanced welding algorithm (much faster, lots more goodies, more arcs produced, better accuracy, firmware compensation for FW that doesn't play well with small arcs, etc...).  I believe most of these are fixed now.  I also had some emergency Octolapse maintenance to do (OctoPrint 1.5 broke my plugin, though it was my fault for not testing the RCs earlier), and have been getting absolutely slammed with issues.  This the season, lol!

Anyway, if you want, you are welcome to test drive the new version and see if it solves your issue.  It is a devel version, so you may have to roll back if you have issues.  It's possible you could lose prints over this, so keep that in mind.  You can always roll back if you need too.

I believe I have 3 testers + myself ATM.  One more (you) would be 20% better :)

Here's a link if you're interested:  https://github.com/FormerLurker/ArcWelderPlugin/archive/8964fbe92181139a4a191cb4137b6644f7985bef.zip

Yuu can install from within the plugin manger by clicking 'get more' and pasting the above address into the 'from url...' box.  Let me know!
